public/js/app.js

```diff
@@ -142,13 +142,19 @@ var makeComment = function(evt){
 
           if(lastComment.length < 1){
             newComment =
-              '<div><h4>1 Responses</h4>'
+              '<div><h4>1 Response</h4>'
             + newComment
             + '</div>';
             form.parent().prev().after(newComment);
+            document.getElementById('comments-project-' + respData.project_id)
+              .innerHTML = 1;
           }
           else {
             lastComment.after(newComment);
+            form.parent().parent().find('h4')
+              .text(respData.commentCount + ' Responses');
+            document.getElementById('comments-project-' + respData.project_id)
```

Comment:
:green_heart:  the way you give a detailed explanation to every PR you submit.

Awesome. Keep it up!